nvm.psm1

```diff
@@ -109,11 +109,6 @@ function Set-NodeVersion {
         $requestedVersion
     }
 
-    if (!(Test-Path -Path $requestedVersion)) {
-        "Could not find node version $matchedVersion"
-        return
-    }
```

Comment:
What’s the reason to remove this?

Reply:
Have a read of the commit message for 2323f82049a6755820f29bf75ad5a2e38d322615, it explains it